csrc/scheduler/matmul.cpp

```diff
@@ -86,8 +85,8 @@ void prologSwizzle(TensorView* shared_mem_tv, const MatmulParams& params) {
   check_concrete_static_dim(shared_mem_tv->axis(-1));
 
   // Extract the constant sizes of the swizzled tile
-  const auto tile_size_x = shared_mem_tv->axis(-2)->extent()->evaluateInt();
-  const auto tile_size_y = shared_mem_tv->axis(-1)->extent()->evaluateInt();
+  const int tile_size_x = (int)shared_mem_tv->axis(-2)->extent()->evaluateInt();
```

Comment:
This has been changed due to compiler's complains?
There were some warnings about used types earlier, but this should be now resolved.